crates/libcontainer/src/syscall/linux.rs

```diff
@@ -28,13 +28,15 @@ use crate::capabilities;
 pub struct LinuxSyscall;
 
 impl LinuxSyscall {
+    #[cfg_attr(coverage, no_coverage)]
```

Comment:
Can't we just apply this to the whole file?

Reply:
@utam0k I tried in the last commit, seems no use.
#[cfg_attr(coverage, no_coverage)]
this mark seems to need wrapping all functions to avoid being counted into coverage.

please guide me if I am doing anything wrong.
I'll reverse the last commit if there is really no way to achieve the goal. 🙇🏻